test/unit/usergroup_test.rb

```diff
@@ -100,8 +92,8 @@ def populate_usergroups
   end
 
   test "removes user join model records" do
-    ug1 = Usergroup.find_or_create_by_name :name => "ug1"
-    u1  = User.find_or_create_by_login :login => "u1", :mail => "u1@someware.com", :auth_source => auth_sources(:one)
+    ug1 = Usergroup.where(:name => "ug1").first_or_create
+    u1  = FactoryGirl.build(:user)
```

Comment:
Oh, disregard.. it's not running the populate method.  It's building the user, then setting #users= is causing u1 to be saved, then UsergroupMember destroyed again.  I think that's OK.